__tests__/app.test.js

```diff
@@ -76,4 +76,26 @@ describe('GET /api/articles/:article_id', () => {
             expect(response.body.msg).toBe('bad request')
         })
     })
+})
+
+describe('GET /api/articles', () => {
+    test('returns a 200 status code and an array of articles sorted by date in descending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toHaveLength(13)
+            expect(body.articles).toBeSorted({ descending: true })
```

Comment:
Jest-sorted requires a key to be specified to know what it's supposed to be sorted by
From the docs:
`expect([{ id: 1 }, { id: 2 }, { id: 3 }]).toBeSorted({ key: 'id' });`